### DISCUSSION TITLE
Unfollow user not working

### DISCUSSION THREAD

**Discussion by seFausto** [community/community#21579] [New to GitHub]
<p>I started following a developer, but after a while, I decided to unfollow them. But, I still see their activity on my feed. I tried turning off any ad block extensions, I tried following, then unfollowing again.</p>
<p>This is nothing against the developer, I just decided to unfollow him.</p>

**Comment by seFausto** [comment, marked as answer]
<p>Nevermind. I realized I had a misunderstanding on how the feature works. If you follow someone and they have activity on day one, then on day 2 you unfollow them, the day 1 activity will still be displayed.</p>

**Comment by RobertoFDuran** [comment]
Has there been any update on how to clear the past activity of someone you unfollowed?

**Reply by seFausto** [reply]
I haven't heard anything. Apparently whatever shows on your feed is stored, even if you unfollow someone. Basically, you stop receiving activities from them, but doesn't clear what has already been displayed.

**Comment by olfek** [comment]
This feature isn't working as expected then.

Most people will expect all activity from the unfollowed persons to disappear,
simply because we've become familiar with this feature and the way it works
on social media sites.

Can someone from GitHub take a look at this please.

**Reply by olfek** [reply]
hi @courtneycl , could you send this to the right people pls, thanks 🙂
